jobs/e2e_node/containerd/k8s-release-branches/image-config-1.30.yaml

```diff
@@ -1,5 +0,0 @@
-images:
```

Comment:
sorry for not caching before. It looks like 1.30 is supported another 20 days: https://kubernetes.io/releases/#release-v1-30

Maybe we should not remove it just yet